src/scripts.js

```diff
@@ -48,14 +49,22 @@ function validateLogin() {
   event.preventDefault();
   let userName = userNameInput.value;
   let password = passwordInput.value;
+  passwordInput.value = '';
+  userNameInput.value = '';
   if (userName.includes('customer')) {
     fetchCustomer(userName.split('r')[1])
     .then(data => {
       let customer = new Customer(data)
       validatePassword(customer, password);
     })
     .catch(error => {
-      console.log(error);
+      console.log('error')
+      let message = `Error please try logging in again!`
```

Comment:
I got this message when using correct credentials when I turned the api off. Could be more specific and let the user know it wasn't a password validation issue.

Reply:
Yes I agree, I'll make that more specific!